bin/before_install

```diff
@@ -13,6 +13,21 @@ if [ -n "$CI" ]; then
   # Install gettext for I18n msgfmt specs
   sudo apt-get install -y gettext
   echo
+
+  if [ -n "$ACT" ]; then
+    # Install cmake so we can build rugged gem
+    sudo apt-get install -y cmake
+    echo
+  fi
```

Comment:
Should this just go in the `if [ -n "$CI" ]; then` conditional above?

Also, is `$ACT` a GHA thing?  I don't see it in https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
$ACT is defined when running locally with https://github.com/nektos/act.  act environments tend to be smaller and not include everything so sometimes we need to install more.  When $ACT is defined, $CI is also defined, so it being nested inside the $CI check is correct.